src/lib.rs

```diff
@@ -135,7 +136,9 @@ impl IOSurface {
                                                   mem::transmute(self.as_concrete_TypeRef()),
                                                   0);
 
-            assert_eq!(gl_error, kCGLNoError);
+            if gl_error != kCGLNoErrer {
+                panic!(&*CStr::from_ptr(CGLErrorString(gl_error)).to_string_lossy());
```

Comment:
I mean leaving this code as is. rust-layers already has an assertion, so right before fatally asserting we could fatally panic instead.